packages/web/src/components/track/GiantArtwork.tsx

```diff
@@ -50,4 +53,41 @@ const GiantArtwork = ({
   )
 }
 
+const SsrGiantArtwork = ({ coSign, artwork }: GiantArtworkProps) => {
```

Comment:
ideally we can use the existing GiantArtwork, with it's cosign wrapping, and just update the DynamicImage/StaticImage part?